src/pages/tokenDrop/TokenDrop.tsx

```diff
@@ -64,18 +64,24 @@ const TokenDrop: FunctionComponent = () => {
         justifyContent: "space-between",
       } }
     >
-      <Box
-        sx={ {
-          display: ["none", "none", "flex"],
-          position: "fixed",
-          top: 32,
-          right: [0, 0, 32, 160],
-        } }
-      >
-        <a href="https://www.newm.io" target="_blank" rel="noopener noreferrer">
-          <NEWMLogo width={ 110 } height={ 110 } />
-        </a>
-      </Box>
+      { isXLargeScreen ? (
```

Comment:
If this wasn't included and used only breakpoints between the two Logo boxes, would run into issues where the mobile view NEWM logo would lose the gradient color circle.